src/tests/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 find_package(Magnum REQUIRED DebugTools AnyImageConverter AnySceneImporter)
 find_package(
   MagnumPlugins REQUIRED GltfImporter StbImageImporter KtxImporter
-  OPTIONAL_COMPONENTS GltfSceneConverter KtxImageConverter
+  OPTIONAL_COMPONENTS GltfSceneConverter KtxImageConverter StanfordImporter
```

Comment:
Can you add `OpenExrImporter`, `OpenExrImageConverter` here, and `AnyImageImporter` to the `find_package(Magnum)` above? While it has no effect for the default build with Magnum bundled, the build with external Magnum would break without it.